var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -11,7 +11,13 @@ class Subversion(AutotoolsPackage):
 
     homepage = 'https://subversion.apache.org/'
     url      = 'https://archive.apache.org/dist/subversion/subversion-1.12.2.tar.gz'
+    urls = [
```

Comment:
You also need to remove this:
```suggestion
    urls = [
```
as a package can either have `urls` or `url`